daphne_worker_test/tests/e2e/test_runner.rs

```diff
@@ -254,6 +254,7 @@ impl TestRunner {
             res.status, res.error
         );
 
+        println!("############ starting test ############");
```

Comment:
this println is intentional. It makes parsing the test output a lot easier by separating logs produced by the initialization of the test from logs produced by the test itself